pom.xml

```diff
@@ -34,7 +34,7 @@
     <version.hdrHistogram>2.1.9</version.hdrHistogram>
     <version.jakarta.enterprise.concurrent-api>1.1.2</version.jakarta.enterprise.concurrent-api>
     <version.jakarta.interceptor-api>1.2.5</version.jakarta.interceptor-api>
-    <version.mp-fault-tolerance>2.0.3</version.mp-fault-tolerance>
+    <version.mp-fault-tolerance>2.1-RC2</version.mp-fault-tolerance>
```

Comment:
2.1 final is now out: https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/2.1

Reply:
Right. I'm just testing one solution for the request context activation in presence/absence of Context Propagation, if it works, this PR will be ready for merging today :-)

Reply:
Great, just don't forget to branch before that :-) if that's still the plan.

Reply:
Why branch before release?

Reply:
From what I understood, SR FT 4.0.x is already used in Quarkus following FT 2.0 spec. Thus a SR FT 4.1.x would be following the FT 2.1 spec.

Reply:
Ok, but why is a branch needed?

I don't believe anything other than Quarkus is using 4.0.x, so we don't need to retain it for anything

Reply:
Yea, we don't need a branch for 4.0 at this point in time. We can always create one retroactively, if there's a need.

Reply:
In general what we do in MW is branches are created to support different versions used in different projects. So in this instance, in order to support that Quarkus version using the SR FT 4.0.0 a "4.0.x" branch is created. If you think there will never be a 4.0.1 then certainly branch is not necessary.

Reply:
For SmallRye we've taken the opposite approach, to only create branches when needed.

The reason is that most often a new release comes out and is used in downstream projects before any updates to an older release would be needed.

In this particular case, as Quarkus will update to the new 4.1.x release, we don't need a 4.0.x branch

Reply:
Ok, although it's confusing and might fly for projects like Quarkus, but not branches that will be used in EAP given it is supported for a long time.

Reply:
Any SmallRye implementation used in EAP, won't be supported for the same length as regular EAP.

It's something we still need to figure out